src/aliased_buffer.h

```diff
@@ -224,8 +226,10 @@ class AliasedBufferBase {
     const size_t new_size_in_bytes = sizeof(NativeT) * new_capacity;
 
     // allocate v8 new ArrayBuffer
+    std::unique_ptr<v8::BackingStore> backing =
+        v8::ArrayBuffer::NewBackingStore(isolate_, new_size_in_bytes);
     v8::Local<v8::ArrayBuffer> ab = v8::ArrayBuffer::New(
-        isolate_, new_size_in_bytes);
+        isolate_, std::move(backing));
```

Comment:
This looks good code-wise, but I don’t think it’s necessary to make these changes? The version that takes a `void*` and a `size_t` is being deprecated, but not the one taking just a `size_t`.

Reply:
Oh you're right. I'll revert itl